parser.cs

```diff
@@ -205,6 +206,7 @@ public class Parser {
                 TYPES typeRec = type();
                 SymbolTable top = __symbolTableStack.Peek();
                 if(typeRec != TYPES.NONE) {
+                    top.IncSize(); // Make room for PC
```

Comment:
When we start processing vars, we need to leave an empty offset between the params and the vars